api/businessservice.go

```diff
@@ -171,7 +171,7 @@ type BusinessService struct {
 	Resource
 	Name        string `json:"name" binding:"required"`
 	Description string `json:"description"`
-	Owner       *Ref   `json:"owner"`
+	Stakeholder *Ref   `json:"owner"`
```

Comment:
This should have been cleaned up a long time ago.  It's left-over from tackle 1.x schema.